onnxruntime/python/tools/transformers/shape_infer_helper.py

```diff
@@ -0,0 +1,62 @@
+import os
+import sys
+
+sys.path.append(os.path.join(os.path.dirname(__file__), '..'))
+from symbolic_shape_infer import *
+
+
+class SymbolicShapeInferenceHelper(SymbolicShapeInference):
+    def __init__(self,
+                 model,
+                 dynamic_axis_mapping={
```

Comment:
Probably not need dynamic axis mapping to compare. If a dimension is symbolic, just compare the string.

Reply:
I found that if not telling the actually value of dynamic axis, the infer shape will fail on most hugginface models